### ISSUE TITLE
phatyp does not run standalone

### ISSUE THREAD

**Issue by bernt-matthias** [KennthShang/PhaBOX#36]
```
(__phabox@2.1.0) ➜  working phabox2 --task phatyp --dbdir '/home/berntm/projects/mb-galaxy-tools/tools/phabox/test-data/phabox_db_v2/' --outpth output/ --contigs example_contigs.fa  --threads 1
PhaBOX2 is running with: 1 threads!
Running program: PhaTYP (Lifestyle prediction)
running with cpu
[1/7] filtering the length of contigs...
[2/7] calling genes with prodigal...
[3/7] running all-against-all alignment...
[4/7] converting sequences to sentences for language model...
[5/7] Predicting the lifestyle...
Map: 100%|██████████████████████████████████████████████████████████████████████████████████████| 351/351 [00:00<00:00, 11357.91 examples/s]
[6/7] summarizing the results...
[7/7] writing the results...
sed: can't read output//final_prediction//cherry_supplementary/alignment_results.tab: No such file or directory
Traceback (most recent call last):
  File "/home/berntm/miniforge3/envs/__phabox@2.1.0/bin/phabox2", line 10, in <module>
    sys.exit(main())
  File "/home/berntm/miniforge3/envs/__phabox@2.1.0/lib/python3.10/site-packages/phabox2/phabox2.py", line 366, in main
    phatyp.run(inputs)
  File "/home/berntm/miniforge3/envs/__phabox@2.1.0/lib/python3.10/site-packages/phabox2/phatyp.py", line 247, in run
    with open(f'{rootpth}/{out_dir}/cherry_supplementary/gene_annotation.tsv', 'w') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'output//final_prediction//cherry_supplementary/gene_annotation.tsv'
```

**Comment by KennthShang** [comment]
This problem is fixed in the 2.1.3 version. maybe you should update the script.

BTW, I am still working on the bioconda scripts [#51830](https://github.com/bioconda/bioconda-recipes/pull/51830). So, maybe you can install it by downloading the latest release and run `python -m pip install . -vv`

Best,
Jiayu

**Comment by KennthShang** [comment]
It seems I cannot update the bioconda scripts timely due to its process. The more reliable channel is my private anaconda account. So install phabox2 with `conda create -n phabox2 phabox=2.1.3 -c jyshang2 -c bioconda -c conda-forge -y` might be a better choice. 

Also, all updates and logs can be found via WIKI [Updates](https://github.com/KennthShang/PhaBOX/wiki/Update-logs)
